packages/create-sitecore-jss/src/templates/vue/data/dictionary/{{language}}.yml

```diff
@@ -0,0 +1,7 @@
+styleguide-sample: Dette er en ordbogspost på dansk som en demonstration
```

Comment:
See other initializers for this file, we don't want to reference Danish anymore.